src/test/java/seedu/address/model/widget/WidgetModelManagerTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.widget;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.address.testutil.TypicalClients.AMY;
+import static seedu.address.testutil.TypicalClients.BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.client.Client;
+import seedu.address.testutil.ClientBuilder;
+
+public class WidgetModelManagerTest {
+
+    @Test
+    void test_setContent_sameClient() {
+        WidgetModel widgetModelManager1 = WidgetModel.initWidget();
+        WidgetModel widgetModelManager2 = WidgetModel.initWidget();
+
+        // set content with same object -> widget object should be the same
+        Client bob = new ClientBuilder(BOB).withTags(VALID_TAG_HUSBAND).build();
+        widgetModelManager1.setContent(bob);
+        widgetModelManager2.setContent(bob);
+        assertEquals(widgetModelManager1.getWidgetContent(), widgetModelManager2.getWidgetContent());
+
+
+        // set content to different client objects but of the same fields and values -> meaning same person but
+        // duplicate entry
+        Client bob1 = new ClientBuilder(BOB).withTags(VALID_TAG_HUSBAND).build();
+        widgetModelManager2.setContent(bob1);
+        assertEquals(widgetModelManager1.getWidgetContent(), widgetModelManager2.getWidgetContent());
+
+    }
+
+    @Test
+    void test_setContent_diffClient() {
```

Comment:
test functions should follow the format: featureUnderTest_testScenario_expectedBehavior()

e.g. sortList_emptyList_exceptionThrown() getMember_memberNotFount_nullReturned

Reply:
Also, probably should make all the tests public void to standardize

Reply:
Changed and fixed checkystle!